config/seata-config-core/src/main/java/io/seata/config/ConfigurationFactory.java

```diff
@@ -31,25 +31,21 @@
     private static final String REGISTRY_CONF_SUFFIX = ".conf";
     private static final String ENV_SYSTEM_KEY = "SEATA_CONFIG_ENV";
```

Comment:
how about `seata_env`?

Reply:
> how about `seata_env`?

Sure,i will unity the name.

Reply:
> how about `seata_env`?

it's done.I will update wiki when this PR merged.